upstairs/src/test.rs

```diff
@@ -5994,7 +5994,7 @@ mod up_test {
         // ----|-------|-----
         //   0 | W     |
         //   1 | FFFFF | 0
-        //   2 | W     | 1
+        //   2 | W     | 0,1
```

Comment:
This, like the example above, I think was still going to be okay.

0 can go when it's ready, as it has no dependencies.
1 can't run until 0 is finished.
2 can't go until 1 has finished, so also requiring 0 here does not change anything.

Reply:
Right, but it's only like this because we don't elide dependencies yet :)